### PR TITLE
GSoC: Adding Week 9 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
image

### CODE REVIEW

File: docs/source/posts/2023/2023-07-29-week-9-praneeth.rst [hunk 1]
```diff
@@ -12,6 +12,10 @@ Continuing from the previous week, it seemed like we were almost done with the *
 
 The default behavior of *TextBox2D* now includes a dynamic bounding box, which scales automatically based on the contained text. Users can customize this option through a simple flag setting. However, this change affected some UI elements like *Combobox2d*, which relied on the default textbox size. Consequently, I had to make updates to ensure compatibility. Additionally, the default initialization of the *TextBlock2D* was completely static, which led to the possibility of the text extending beyond the background and failing certain tests. To tackle this, I made adjustments to the overflow helper function in the *test_elements.py* file. After a few tweaks and issue resolutions, the PR was ready for review and was successfully merged after passing the review process.
```
Comment (JoaoDell) [inline docs/source/posts/2023/2023-07-29-week-9-praneeth.rst:13]: *`Combobox2d`* -> *`ComboBox2D`*

**Review by JoaoDell — COMMENTED** [review]
Hey @ganimtron-10, happy to see your progress! Your blogpost seems ok, just check a minor comment I left below 👍

File: docs/source/posts/2023/2023-07-29-week-9-praneeth.rst (hunk 1 above)
Comment (ganimtron-10) [reply docs/source/posts/2023/2023-07-29-week-9-praneeth.rst:13]: https://fury.gl/latest/reference/fury.ui.html#combobox2d

The UI mentions it this way only ie. `ComboBox2D` that why I mentioned it that way!

Comment (JoaoDell) [reply docs/source/posts/2023/2023-07-29-week-9-praneeth.rst:13]: Got it, its fine

**Review by skoudoro — APPROVED** [review]
I was waiting for @tvcastillod review. 

but all good so I am going ahead and merge this PR

**Pull request merged**